### ISSUE TITLE
Allow multiple comma-separated email recipients in Jet Forms "Send Email" action

### ISSUE THREAD

**Issue by git-temak** [CrocoBlock/suggestions#4443]
JetEngine Forms and JetFormBuilder are missing one vital feature in the post-submit action "Send Email". Admin is unable to set up a form where you can easily enter multiple email addresses to receive form submission notifications - http://prnt.sc/219dh3l. At the moment, you need to manually create multiple "Send Email" actions to achieve this option - https://prnt.sc/218ho9x. A number of other form plugins such as CF7, superforms, etc. all have the ability to add all email recipients in the "send to" email field and separate each email with a comma - https://prnt.sc/219drvd.

It would save a lot of unnecessary duplication if this is implemented. The actions cannot also be automatically duplicated since they essentially have the same information and vary by just the email field so one needs to manually type in or copy/paste the actions over and over for each email recipient.

Kindly help work on this.

**Comment by spingoo** [comment]
I agree. This is an important feature that can save a lot of work when dealing with sending emails.

**Comment by Crocoblock** [comment]
![Image](https://github.com/user-attachments/assets/80384945-8cc9-4ebc-b08e-98da515c4542)
